docs/kubefirst/local/install.md

```diff
@@ -37,6 +37,14 @@ The `kubefirst local` command will request that you authorize the Kubefirst CLI
 
 The [gitops repo](https://github.com/kubefirst/gitops-template) that we create for you will power your local Kubefirst platform. The [metaphor-frontend](https://github.com/kubefirst/metaphor-frontend-template) repo is your microservice example, which demonstrate how to publish and gitops-deliver applications to your new development, staging, and production namespaces in your new local cluster.
 
+#### Atlantis and Ngrok integration
+
+During cluster provisioning, Terraform communicates with host machine to create the desired resources. When Atlantis is installed via Kubefirst, it will use Ngrok to expose the Atlantis server to the internet via webhook.
```

Comment:
During cluster provisioning, Terraform communicates with the host machine to create the desired resources. When Atlantis is installed via Kubefirst, it will use ngrok to expose the Atlantis server to the internet via webhook.

Webhooks are the way GitHub listen to events, and forward the request to a service. As a example, when a new commit is pushed to a repository, Atlantis will receive an event call, and run a plan.

ngrok is a tool that allows us to expose a local server to the internet via a tunnel, Kubefirst opens the tunnel during the installation, and Atlantis receive the events via the tunnel. When the installation finish, it hangs at the handoff screen. The tunnel will be closed once the handoff screen is closed, and the Kubefirst installation process dies.